creator-node/src/services/stateMachineManager/stateReconciliation/recoverOrphanedData.jobProcessor.ts

```diff
@@ -245,23 +246,28 @@ const _batchIssueReqsToRecoverOrphanedData = async (
   return requestsIssued
 }
 
-// Primary could potentially be passed from state machine, but the added complexity isn't worth it.
```

Comment:
did you mean to remove this comment

Reply:
yeah I figured it's just clutter if it's not high pri enough to resolve. also we have that ticket you opened with platform to batch fetching multiple wallets